examples/cart/styles/_overlay.scss

```diff
@@ -0,0 +1,12 @@
+.overlay {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: none;
+
+  &.js-active {
+    display: block;
```

Comment:
the overlay is transparent, just there to act as a big event handler for clicking away from the cart and providing a place for customizing the overlay if wanted